app/retail/emails.py

```diff
@@ -1021,14 +1021,87 @@ def render_new_bounty_roundup(to_email):
 </p>
     '''
 
-    intro = args.body.replace('KUDOS_INPUT_HERE', kudos_friday)
-    highlights = args.highlights
-    sponsor = args.sponsor
-    bounties_spec = args.bounties_spec
+    intro = intro = f'''
```

Comment:
We now draw this content via the database, so the placeholder content here is not needed

Reply:
I shall remove this too